src/test/lit-html_test.ts

```diff
@@ -162,6 +161,54 @@ describe('grimlock', () => {
               `);
       });
 
+      it('unbound boolean attributes', () => {
+        expect(
+          grimlock.convertModule(
+            'test.ts',
+            js`
+              import {html} from 'lit-html';
+
+              /**
+               * @soyCompatible
+               */
+              export const t = () => {
+                return html\`<input disabled>\`
+              };
+            `
+          ).files[0].content
+        ).toEqual(soy`
+          {namespace test.ts}
+
+          {template .t}
+          <input disabled="">
```

Comment:
Why isn't this just `<input disabled>`?

Reply:
p5's `node.attrs` gives `{name: "input", value: ""}` for boolean attributes.

I think to nix the `=""`, we would require a list of every boolean attribute. Seems not worth it since `disabled=""` is just as valid, even though that'd be nice.

Reply:
There will be a list of all built-in boolean attributes somewhere. I've had to dig this info up for tools before, I think by scraping MDN. Something like this might help: https://github.com/extensibleweb/webidl.js